src/main/java/seedu/address/model/person/SearchContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
```

Comment:
Don't think the header matches the method anymore, remember to change headers when you refactor / change methods